decimal_test.go

```diff
@@ -156,13 +166,16 @@ func TestNumberUnmarshalText(t *testing.T) {
 func TestNumberScan(t *testing.T) {
 	var a Number
 	assert.Nil(t, a.Scan([]byte("0.015")))
-	assert.Equal(t, Number{15, -3}, a)
+	assert.Equal(t, newDecimal.New(15, -3), a)
 
 	assert.NotNil(t, a.Scan("Strings are not supported"))
 }
 
+// FIXME: this needs discussion, output changed:
```

Comment:
Does this break anything?

Reply:
Test fail so I guess it will break something.

Reply:
Dovydas PR if passed will fix this https://github.com/shopspring/decimal/pull/288/files. 
At the moment `NewFromRat` was ported from https://github.com/shopspring/decimal/pull/4/files which works for small Rats, big.Rats won't necessarily fit into a Float64, so we will be losing information here.

Reply:
I don't think that is is significant as this should work either way for us when scanning. We don't use driver's value anywhere, I think